File: JavaScript/1015.js
var input = require('fs').readFileSync('stdin', 'utf8');

var valores = input.split("\n");
var [x1, y1] = valores[0].split(" ");
var [x2, y2] = valores[1].split(" ");

var distancia = Math.sqrt(Math.pow((parseFloat(x2)-parseFloat(x1)),2.0)+Math.pow((parseFloat(y2)-parseFloat(y1)),2.0));

console.log((distancia).toFixed(4));

File: C/1015.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main() {

    double xa, ya, xb, yb;
    scanf("%lf", &xa);
    scanf("%lf", &ya);
    scanf("%lf", &xb);
    scanf("%lf", &yb);
    printf("%.4lf\n", sqrt(pow(xb-xa, 2.0) + pow(yb-ya, 2.0)));

    return 0;
}